src/components/FileList.tsx

```diff
@@ -130,6 +127,43 @@ export class FileList extends React.Component<IFileListProps, IFileListState> {
         }
       });
     }
+
+    if (!commands.hasCommand(CommandIDs.gitFileDiffWorking)) {
```

Comment:
Why are we doing this conditionally?

Reply:
I kept it consistent with what we have in this file. I tested out by removing all the conditionals in this file and didn't see any issues on local testing, so went ahead and removed them all.